avm/res/purview/account/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,46 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-purview-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = 'eastus' // Only available in selected locations: eastus, eastus2, southcentralus, westcentralus, westus, westus2, westus3
```

Comment:
@krbar this value would be overwritten by the CI. If you' d want to hardcode it, it would need to be hardcoded in the deployment below.

In any case, like the storage PR #1052, this is another one where the general locations would run into trouble @eriqua & @ChrisSidebotham. I guess that means the PR would be blocked?

Reply:
Same for the other test files of course.

Reply:
Good catch, thanks. The hardcoded location comes from CARML. I don't insist on hardcoding it if we can ensure in other ways that only defined (supported) regions are used. We would probably need something like "allowedRegions" that can be defined per resource type, either as a parameter, or determined dynamically.

For now, I'll move the hardcoded location down to the deployment, but I'm open for better ideas.

Reply:
Also in this case I'd suggest to double check if the default location list covers this

Reply:
The default location list is: 'eastus', 'uksouth', 'northeurope', 'eastasia'

In other words, the deployment will fail for 
- `uksouth`
- `northeurope`
- `eastasia`

and work if `eastus` is selected
cc: @eriqa

Reply:
In the mean time much more regions are supported:
`eastus`, `westeurope`, `southeastasia`, `canadacentral`, `canadaeast`, `swedencentral`, `southcentralus`, `brazilsouth`, `centralindia`, `uksouth`, `francecentral`, `koreacentral`, `uaenorth`, `japaneast`, `switzerlandnorth`, `westus`, `southafricanorth`, `germanywestcentral`, `qatarcentral`, `westus3`, `australiaeast`, `northeurope`, `westcentralus`, `westus2`, `eastus2`

However `eastasia` is not on the list.

Reply:
> @krbar this value would be overwritten by the CI. If you' d want to hardcode it, it would need to be hardcoded in the deployment below.

Placing a hardcoded location directly in the deployment creates a Bicep warning and an issue while generating the README. Therefore I've created a new parameter `fixedResourceLocation` to overcome these issues. I still hope we can remove this hardcoded location completely.

Reply:
See comment I added

Reply:
The region rotation is anyway selecting the location among the default locations which are also available for the service. If eastasia is not in the list of available locations for purview, it won't be picked. Did we test the module with the default behavior?

Reply:
No, this is new to me. I can give it a try.

Reply:
It looks like the region selection logic works fine for the Purview. I ran the pipeline several times in a row and there were no attempts to deploy in `eastasia`. I removed the workaround.